.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,6 +8,7 @@ UCD-
 ## Checklist
 <!-- Put an `x` in all the boxes that apply: -->
 - [ ] Requires infrastructure changes
+  - [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate helm chart
```

Comment:
Can this be formatted as lines above/below? It looks squiffy when viewed in GitHub

Also, "Helm" with capital H

Reply:
Awww maaaan, it was deliberately nested under `Requires infrastructure changes`

No worries ;)

Reply:
Aaaah, sorry. It didn't seem to put the bullet points when I viewed it